setup.py

```diff
@@ -69,14 +83,81 @@ def run(self):
+        os.mkdir(dest_dir)
+      # For development mode (pip install -e .) gfootball_engine module has to be located
+      # in the project root directory. So we copy only __init__.py and `data` directory
+      shutil.copy2('third_party/gfootball_engine/__init__.py', dest_dir)
+      data_dir = os.path.join(dest_dir, 'data')
+      if not os.path.exists(data_dir):
+        shutil.copytree('third_party/gfootball_engine/data', data_dir)
+
+    # Copy fonts
+    dst_fonts = os.path.join(dest_dir, "fonts")
+    if not os.path.exists(dst_fonts):
+      shutil.copytree("third_party/fonts", dst_fonts)
+
+    py_major, py_minor, _ = platform.python_version_tuple()
+
+    if download_engine:
+      # TODO: Automate wheels creation for Windows, until then download from GitHub releases and unzip dlls
+      if py_major != '3' or py_minor not in ('7', '8', '9'):
+        raise OSError("Unsupported Python version. Try compiling engine instead. " 
+                      "Precompiled binaries are available for Python 3.7-3.9 only")
+      import urllib.request
+      import urllib.error
+      import zipfile
+      import io
+      bit_ver = '64' if sys.maxsize > 2 ** 32 else '32'
+      download_link = 'https://github.com/vi3itor/football/' + \
+                      f'releases/download/v{VERSION}/' + \
+                      f'gfootball-engine-win-py{py_major}{py_minor}-{bit_ver}.zip'
+      try:
+        with urllib.request.urlopen(download_link) as response, \
+            zipfile.ZipFile(io.BytesIO(response.read())) as bin_zip:
+          bin_zip.extractall(dest_dir)
+      except urllib.error.HTTPError:
+        raise Exception("There's a problem with downloading precompiled gfootball engine")
+      except:
+        raise Exception("Unable to download precompiled gfootball engine")
+    else:  # compile the engine
+      # help message
+      guide_message = 'Please follow the guide on how to install prerequisites: ' \
+                      'https://github.com/vi3itor/football/blob/windows/gfootball/doc/compile_engine.md#windows'
+      if not os.environ.get('VCPKG_ROOT'):
+        raise OSError('VCPKG_ROOT environment variable is not defined.\n' + guide_message)
+      os.environ['GENERATOR_PLATFORM'] = 'x64' if sys.maxsize > 2 ** 32 else 'Win32'
+      os.environ['PY_VERSION'] = f'{py_major}.{py_minor}'
+      if os.system('gfootball\\build_game_engine.bat'):
+        raise OSError('Google Research Football compilation failed.\n' + guide_message)
+      # Copy compiled library and its dependencies
+      lib_path = 'third_party/gfootball_engine/build_win/Release/'
+      libs = glob.glob(f'{lib_path}*.pyd') + glob.glob(f'{lib_path}*.dll')
+      for file in libs:
+        shutil.copy2(file, dest_dir)
+
+
+# To support development (a.k.a. editable) install (`pip install -e .` or `python3 setup.py develop`),
+# we remove `build` directory created by a regular setup
+if 'develop' in sys.argv and os.path.exists('build'):
+  shutil.rmtree('build')
```

Comment:
If a `build` directory was already created, e.g. by running `pip install .` previously, then it will be used by the development install. In such a case, import of `gfootball_engine` will fail, since the the package is not located in the project root.